source/reduced.js

```diff
@@ -7,7 +7,7 @@ import _reduced from './internal/_reduced';
  * box: the internal structure is not guaranteed to be stable.
  *
  * Note: this optimization is only available to the below functions:
- * - [`reduce`](#reduce)
+ * - [`reduce`](#reduce) (**when passed a transformer**)
  * - [`reduceWhile`](#reduceWhile)
```

Comment:
Will `reduced` still work on `reduceWhile`? I would think we should take it out there too. Thoughts?

Reply:
On a second read through, it appears `reduceWhile` depends on `reduced`, but I don't necessarily think we should advertise that fact.